### ISSUE TITLE
Proposal for limited scope of GA4GH Variant Representation

### ISSUE THREAD

**Issue by larrybabb** [ga4gh-gks/variant-representation-model#1]
Added by lbabb on behalf of eric moyer...
---
From: Eric Moyer 
Date: Monday, May 14, 2018 at 1:48 PM
Subject: Proposal for limited scope of GA4GH Variant Representation

There are many, many things that different groups want to say about variants. If we support all of them, we will have a standard that is so big that no one will implement it. The standard can grow. But that has to happen after people adopt it. HTML is currently a behemoth. But it started with a document with less than 20 elements and a few pages of descriptive text.

I propose that we serve one market to start with: variant representation in electronic health records.

Why:

- Benefits a large number of people (people who have genetic information and EHR)
- This market is highly regulated so the existence of a standard is a feature highly desired by the market.
- This market touches a lot of areas. If something becomes standard practice there, others will use it too because they need to interact with EHR anyway. This will drive expansion of GA4GH Variant Representation (VR) into other areas.

This could have streamlined our discussion today greatly. For EHR, no one cares that the variant is a translocation. It is just two variants, a deletion from one chromosome and an insertion on another (or on the same one). It doesn’t matter where the other part came from. No one cares whether a variant is an inversion. It is just a change. My patient does not have the history of how the variant came about.

Researchers care about these things. People trying to prevent the mutation in the future care about them. Evolutionary biologists care about them. Population geneticists care about them. But a doctor trying to treat a patient only cares about what is, not the process by which it came to be. A pharmacist prescribing a drug cares what variants a person has, not what sequence they might have had beforehand if the dice of inheritance and environment had rolled differently.

_Minimal additional primitives:_
This focus is useful in itself as a good way of organizing our standardization. However, I conjecture that with this focus we don’t need to worry about joining structural and precise variants. It simplifies things enough that we can accommodate structural variants into VR by adding as few as three new primitives: imprecise endpoints, complex variants, and imprecise sequence. I don’t make this conjecture very strong because I haven’t been discussing things with the structural variant subcommittee.

And with this focus, the “translocation” or “inversion” or “copy number” could be added as an annotation later. It is an interpretation of the history.

Answering some objections to my minimal primitive idea above:
If someone is worried that large variants will be hard to represent. That is a matter of compression. It could be that we will mandate a compression algorithm that can easily use parts of the reference assembly or equivalently we could tack on two additional primitives: “use sequence from this precise region” and “then do this”.

If someone is worried that these will be hard to read, that is the job of the software presenting the variant. Which should present the long variants in a compressed and abstracted manner.

**Comment by andrewyatz** [comment]
I agree with the sentiment laid out by Eric. Especially the point of a standard being too complicated for use before it even lands. Also anything we, GA4GH and NCBI, can do to push for a unified representation is the right way to go for sure. I don’t feel any of us want to write these because we can.

My main concerns would be the HL7/FHIR elephant in the room. Is there space for another variant representation in EHRs with FHIR in place?

I also question what’s the risk to reducing the variant representation as you say? Also the possible groups that may embrace/reject simplification. In my, albeit very limited, experience researchers do care about variation specifics but larger scale data processors are happy to accept reduced representation (see VCF as an example). It’s clinicians where I’ve heard of more reluctance when describing variants and drive use-cases similar to what we see in HGVS.

But to reiterate very happy that we start looking at this and we do it together for sure!

**Comment by mbaudis** [comment]
This is a rather complex proposal which discusses several points, which may not be necessarily related: Scope/target (i.e. representation in EHR), need to reduce complexity (i.e. targeting existence of variant "primitives" instead of complete contextual representation) and then some specific examples (e.g. "translocation as something happened here and there").

#### My POVs, briefly:

* I'm (personally) not comfortable with "targeting EHR"; I see this as a complex and specific area, where there are also lot's of activities which need to be synchronised with - @andrewyatz notes about HL7/FHIR
* the reduction to "primitives" is necessary, but those go a bit beyond the "some event at location"; a limited set of specific changes have to be specified from the start
    * CNV with type are absolutely necessary (though not the quantitative details, ploidy - one has to be able to represent the MYCN amplification...)
    * fusion events need to be represented, either
        - directly (`refChro:pos1` <-> `mateChro:pos2`), or
        - through linking (`refChroA:pos1:break:id_N123`, `refChroB:pos2:break:id_N9876`, `id_N123` <=> `id_N9876`)
   ... where the linking then introduces a requirement beyond "primitives"

I, personally, would like to see traction for work on a model which builds on the concept of VCF ("represent single variants") for single alleles (including fusions, i.e. events not part of the VMC allele model), but leaves the haplotype/genotype composition as out-of-scope / for later.
